packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx

```diff
@@ -14,3 +18,22 @@ export const clientIdTooltipText =
  */
 export const userIdTooltipText =
 	"Represents the application-specific user identifier provided to the Fluid Framework. Auto-generated by Fluid if not specified by the application.";
+
+/**
+ * Description of the Container "Status" property in the summary view.
+ */
+export const containerStatusTooltipText = (
+	<div>
+		Represents the current state of the Container with regards to its connection to the Fluid
+		Service. For more information, see{" "}
+		<Link
```

Comment:
Curious; what do we get from the `Link` component over writing an `<a>` by hand?

Reply:
`Link` ensures we get the Fluent UI theming applied correctly. I initially used `<a>`, but the coloring in dark mode was terrible. There's probably a way to apply the theming explicitly to the `<a>` tag, but this seemed easier.